src/rpc/format/cbor/convert.rs

```diff
@@ -244,6 +268,8 @@ impl TryFrom<Cbor> for Value {
 	}
 }
 
+const SERIALIZATION_V2: bool = false; // 💡 enable and clean code for v2
```

Comment:
I can't make a suggestion for some reason, could you place the comment above and make it
```
// TODO(kearfy): Make this the default in v2.0.0 and drop this boolean
```